static/now.json

```diff
@@ -14,7 +14,7 @@
             "headers": {
                 "cache-control": "s-maxage=31536000,immutable"
             },
-            "dest": "/src/static/$1"
+            "dest": "/src/$1"
```

Comment:
Can we avoid having a src folder here and just have the asset folders? If there's no build pipeline, then I see having a src folder as superfluous / additional nesting. Unless Now needs it in some way? :man_shrugging: 

Also, the css stuff needs to be in the lib so we can import it in parent apps. I had implemented that as part of the build pipeline which spat out the reset.css and styles.css at the project root for easy access, are we not doing this anymore? The problems I was running into were relating to npmignore and files in the package json.  However, my previous PR should have fixed that, until we changed the build config.

So either we need to add `static` to the `files` field in package.json and then import like:

`import '@stylindex/octopus/static/src/css/reset.css'`, which seems a bit verbose.

Or import the reset statically in the app via a `link` tag (slow and will lead to FOUC).

Or, my preferred option, add the css to the build, output it to the directory root when building (obviously having the /reset.css and /styles.css output files in the .gitignore), add the two files to the `files` field so they're part of the package and then import like:

`import '@stylindex/octopus/reset'` or similar.

What do you think?

Reply:
Regarding the src folder, yes this this is the best way to avoid any config files being served

Reply:
should be good now @lesbaa. I've included the css using `styled-components` global